src/main/java/duke/Main.java

```diff
@@ -0,0 +1,128 @@
+package duke;
+
+import javafx.application.Application;
+import javafx.application.Platform;
+import javafx.geometry.Insets;
+import javafx.scene.Scene;
+import javafx.scene.control.Button;
+import javafx.scene.control.Label;
+import javafx.scene.control.ScrollPane;
+import javafx.scene.control.TextField;
+import javafx.scene.image.Image;
+import javafx.scene.image.ImageView;
+import javafx.scene.layout.AnchorPane;
+import javafx.scene.layout.Region;
+import javafx.scene.layout.VBox;
+import javafx.stage.Stage;
+
+/**
+ * Main is where the GUI is initialised and shown
+ */
+public class Main extends Application {
+
+    private Duke duke = new Duke();
+
```

Comment:
perhaps some empty lines could be removed to make the code look more concise. I've noticed this in some other places as well.